ItemLevelDownloader/Providers/localFilesystemProvider.ts

```diff
@@ -0,0 +1,106 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as models from "../Models"
+import * as Stream from "stream";
+
+export class LocalFilesystemProvider implements models.IArtifactProvider {
```

Comment:
removed fileShareProvider  and renamed LocalFiesystemProvider to FilesystemProvider